pkg/tfvars/baremetal/baremetal.go

```diff
@@ -80,10 +83,16 @@ func TFVars(libvirtURI, ironicURI, bootstrapOSImage, externalBridge, provisionin
 		}
 
 		// Instance Info
+		// The rhcos-downloader container downloads the image, compresses it to speed up deployments
+		// and then makes it available on bootstrapProvisioningIP via http
+		imageFilename := path.Base(image)
```

Comment:
Are we doing the same thing as Machine here for bootstrap? why not use the Machine object itself.

Reply:
It's not quite the same, the rhcos-downloader is running in a different location (on the bootstrap VM vs the cluster in the machine case), so we have to use a different IP.

May be this can be a bit DRY-er, but perhaps I can do that as a followup, with some URL generating utility function or something?